tools/percy/snapshots.js

```diff
@@ -1,5 +1,11 @@
 const PercyScript = require("@percy/script");
 const scrollToBottom = require("scroll-to-bottomjs");
+
+// Puppeteer is imported so that Code and friends can provide types.
+const puppeteer = require('puppeteer');  // eslint-disable-line no-unused-vars
+
+const isDev = Boolean(process.env.WEBDEV_PERCY_DEV)  // set by snapshots-dev.js
```

Comment:
Why do you need this? I was able to run Percy locally without needing to set environment variables (though, you do have to export the PERCY_TOKEN first). Also, I think you'll want the `ELEVENTY_ENV=test` to be set if you're doing percy snapshots because that forces the blog to display the same cards https://github.com/GoogleChrome/web.dev/blob/master/src/site/_collections/post-descending.js#L20. Otherwise every new post would cause the snapshot to fail because it would show up on /blog

Reply:
This lets me watch Percy run. I just want to pass a flag letting me see it work.

Reply:
You mean it opens a chrome window and you can see it scrolling around and whatnot?

Reply:
Yup. Is there another way to make Percy run that behavior?

I thought about maybe, if this script is running (rather than being `require()`'d) but it seems like Percy just runs it directly too. I thought maybe it was being imported by a Percy harness, but alas.